compositing-1/Overview.bs

```diff
@@ -14,7 +14,7 @@ Editor: Rik Cabanier, Adobe Systems Inc., cabanier@adobe.com, w3cid 106988
 Editor: Nikos Andronikos, Canon Information Systems Research Australia, Nikos.Andronikos@cisra.canon.com.au
 Abstract: Compositing describes how shapes of different elements are combined into a single image. There are various possible approaches for compositing. Previous versions of SVG and CSS used <a href="https://www.w3.org/TR/SVG11/masking.html#SimpleAlphaBlending">Simple Alpha Compositing</a>. In this model, each element is rendered into its own buffer and is then merged with its <a>backdrop</a> using the Porter Duff ''source-over'' operator. This specification will define a new compositing model that expands upon the Simple Alpha Compositing model by offering:
 Abstract: <ul><li>additional Porter Duff compositing operators</li><li>advanced blending modes which allow control of how colors mix in the areas where shapes overlap</li><li>compositing groups</li></ul>
-Abstract: In addition, this specification will define CSS properties for blending and group isolation and the properties of the <a href="https://www.w3.org/TR/2dcontext2/#dom-context-2d-globalcompositeoperation" title="canvas 2d globalcompositeoperation">‘globalcompositeoperation’</a> attribute as defined in [[2DCONTEXT2]].
+Abstract: In addition, this specification will define CSS properties for blending and group isolation and the properties of the <a href="https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-globalcompositeoperation" title="canvas 2d globalcompositeoperation">‘globalcompositeoperation’</a> attribute as defined in [[2DCONTEXT2]].
```

Comment:
Replace this with `<<canvas/globalCompositeOperation>>`, and delete the biblio reference to the old W3C spec.

Reply:
Done